apps/docs/src/app/platform/component-docs/platform-forms/platform-input/platform-input-example/platform-input-auto-complete-validation-example.component.ts

```diff
@@ -29,32 +30,35 @@ export class PlatformInputAutoCompleteValidationExampleComponent implements OnIn
     @Input()
     open: boolean;
```

Comment:
Do we really need this input? Noone uses it.

Reply:
Fixed.